ltslam:

  sessions_dir: "/home/user/Documents/catkin2021/catkin_LTslam/data_kitti/in/" # end with / 

  central_sess_name: "011" # should be same as the directory name of that session
  query_sess_name: "02" # should be same as the directory name of that session
  
  save_directory: "/home/user/Documents/catkin2021/catkin_LTslam/data_kitti/out/011-02-manyloops/" # end with "/"

  is_display_debug_msgs: false

  loopFitnessScoreThreshold: 0.7

  kNumSCLoopsUpperBound: 1000 # if you use an enough number, use all the detected loops 
  kNumRSLoopsUpperBound: 0 # if you use an enough number, use all the detected loops 

  numberOfCores: 16 # currently hard-coded in the utility.cpp

